### ISSUE TITLE
Tumor type filter not working

### ISSUE THREAD

**Issue by zperova** [PDCMFinder/pdxfinder#857]
**Describe the bug**
Tumor type filter does not work.

**To Reproduce**
1. Go to https://www.pdxfinder.org/data/search
2. Click on Tumor type - any of the options
3. See result - returned 0 results in 0 sources

**Expected behavior**
Choosing specific tumor type should return results.

**Screenshots**

![Screenshot 2022-03-03 at 21.12.39.png](https://images.zenhubusercontent.com/5b631938e115f66327ad7ac6/1129a8d9-91d6-4cee-a38f-37caa01be407)

**Desktop (please complete the following information):**
 - OS: MacOs
 - Browser Chrome
 - Version Version 97.0.4692.71 (Official Build) (x86_64)

**Comment by mauroz77** [comment]
@zperova At some point the values in the tumourType node (neo4j) were store as capitalized values (Primary, Metastatic, etc) and they matched exactly the values of the filters. Now the values in the database are lowercase and the logic was comparing the 2 values as they are (Primary vs primary) so they weren't matching and therefore no results were found.

What I did is to change the logic so the comparison changes first the values to lowercase in both sides (filter and values). In other words, I made the search, when using values of the checkboxes, case unsensitive.
This solution avoids any change in the database.
Does it sound ok?
